wagtail/core/models.py

```diff
@@ -3630,11 +3581,10 @@ def cancel(self, user=None, resume=False, comment=''):
     def copy(self, update_attrs=None, exclude_fields=None):
         """Copy this task state, excluding the attributes in the ``exclude_fields`` list and updating any attributes to values
         specified in the ``update_attrs`` dictionary of ``attribute``: ``new value`` pairs"""
-        copy_instance, _ = self._copy(exclude_fields, update_attrs)
-        return copy_instance
-
-    def _save_copy_instance(self, instance, **kwargs):
+        exclude_fields = self.default_exclude_fields_in_copy + self.exclude_fields_in_copy + (exclude_fields or [])
+        instance, _ = _copy(self.specific, exclude_fields, update_attrs)
```

Comment:
@kaedroho wouldn't the use of `_` conflict with the `from django.utils.translation import gettext_lazy as _` import?

Reply:
oh yes, nice spot!

Reply:
This is on master too: https://github.com/wagtail/wagtail/blob/master/wagtail/core/models.py#L3633